client/actions/planning/api.js

```diff
@@ -300,6 +300,15 @@ const getCriteria = ({
                 must.push({term: {featured: true}});
             },
         },
+        {
+            condition: () => (Array.isArray(advancedSearch.place) &&
```

Comment:
@nrvikas you might have too add place search clause in combined view search endpoint?